custom_components/opnsense/sensor.py

```
@@ -41,322 +42,370 @@
+            if prop_name == "connected_client_count":
+                state_class = SensorStateClass.MEASUREMENT
 
-                if prop_name in ["connected_client_count"]:
-                    native_unit_of_measurement = "clients"
+            # native_unit_of_measurement
+            if "_kilobytes_per_second" in prop_name:
+                native_unit_of_measurement = UnitOfDataRate.KILOBYTES_PER_SECOND
 
-                # icon
+            if native_unit_of_measurement is None:
                 if "bytes" in prop_name:
```

Ruff

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)